pkg/build/strategies/sti/sti.go

```diff
@@ -115,10 +115,12 @@ func (b *STI) Build(config *api.Config) (*api.Result, error) {
 		return nil, err
 	}
 
-	if b.incremental = b.artifacts.Exists(config); b.incremental {
-		glog.V(1).Infof("Existing image for tag %s detected for incremental build", config.Tag)
+	if !config.Incremental {
+		glog.V(1).Infof("Incremental build is disabled. Clean build will be performed")
+	} else if b.incremental = b.artifacts.Exists(config); b.incremental {
+		glog.V(1).Infof("Existing image for tag %s detected. Perform Incremental build with save-artifacts under the %s", config.Tag, b.scriptsURL["save-artifacts"])
```

Comment:
I'm not too happy with those changes, imho the V1 verbosity currently provides enough information about the build process. Especially that the information about whether build has incremental enabled or disabled is already printed at the beginning, see [here](https://github.com/openshift/source-to-image/blob/master/cmd/sti/main.go#L165-L167) where the entire Config is being printed, but at a higher log level - 2. Above changes complicate the code more than needed.